components/brave_new_tab_ui/components/default/cryptoDotCom/index.tsx

```diff
@@ -127,12 +126,12 @@ class CryptoDotCom extends React.PureComponent<Props, State> {
   }
 
   checkSetRefreshInterval = () => {
-    if (!this.refreshInterval) {
-      this.refreshInterval = setInterval(async () => {
-        await this.props.onUpdateActions()
-          .catch((_e) => console.debug('Could not update crypto.com data'))
-      }, 30000)
-    }
+    // if (!this.refreshInterval) {
+    //   this.refreshInterval = setInterval(async () => {
+    //     await this.props.onUpdateActions()
+    //       .catch((_e) => console.debug('Could not update crypto.com data'))
+    //   }, 30000)
+    // }
```

Comment:
This was an accidentally committed debug step. Still debugging, so please ignore comments for now.